web/pgadmin/tools/sqleditor/static/js/components/sections/MainToolBar.jsx

```diff
@@ -562,12 +569,22 @@ export function MainToolBar({containerRef, onFilterClick, onManageMacros, onAddT
           <PgIconButton title={gettext('Help')} icon={<HelpIcon />} onClick={onHelpClick} />
         </PgButtonGroup>
       </StyledBox>
+      <PgMenu
+        anchorRef={openInNewTabMenuRef}
+        open={openMenuName=='menu-openfileintab'}
+        onClose={onMenuClose}
+        label={gettext('Open file Menu')}
+      >
+        <PgMenuItem hasCheck value="open_in_new_tab" checked={checkedMenuItems['open_in_new_tab']}
+          onClick={checkMenuClick}>{gettext('Open in new tab?')}</PgMenuItem>
```

Comment:
Rename 'Open in new tab?' to 'Open in a new tab?'.

Reply:
Renamed to  'Open in a new tab?'.